talend-component-maven-plugin/src/it/web/test/component_runtime_dev.json

```diff
@@ -0,0 +1,1289 @@
+{
+  "version": 6,
+  "entities": [
+    {
+      "entity": {
+        "type": "Project",
+        "description": "To run the test you need to run a component server (1.45 minimum). \nGo to a component directory and run:\nmvn talend-component:web \"-Dtalend.web.batch=true\" \"-Dtalend.web.batch.timeout=1\" \"-Dtalend.web.port=8081\"  \"-Dtalend.web.batch.timeout=5\"\n\nApi documentation is herer https://talend.github.io/component-runtime/main/latest/rest-openapi.html",
```

Comment:
typo in description.

Reply:
This file is generated in API tester, I did the correction.